matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/profile/ProfileService.kt

```diff
@@ -48,6 +49,14 @@ interface ProfileService {
      */
     fun setDisplayName(userId: String, newDisplayName: String, matrixCallback: MatrixCallback<Unit>): Cancelable
 
+    /**
+     * Update the avatar for this user
+     * @param userId the userId to update the avatar of
+     * @param newAvatarUri the new avatar uri of the user
+     * @param fileName the fileName of selected image
+     */
+    fun updateAvatar(userId: String, newAvatarUri: Uri, fileName: String, matrixCallback: MatrixCallback<Unit>): Cancelable
```

Comment:
Can we update avatar of other users? otherwise, we should not offer the userId param and inject @UserId instead

Reply:
The Matrix API take a userId, that's why it's exposed here... But yes in the app, it will be only possible to change avatar of the current user